pytest-embedded-idf/pytest_embedded_idf/serial.py

```diff
@@ -194,6 +194,36 @@ def dump_flash(
         else:
             return content
 
+    @EspSerial.use_esptool
+    def erase_partition(self, partition_name: str = None) -> None:
+        """
+        Erase the partition provided
+        """
```

Comment:
here you may add the param in [google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). this could generate the API doc automatically.

```python
def erase_partition(self, partition_name: str) -> None:
    """
    Erase the partition provided.

    Args:
        partition_name: partition name
    """
```